mmv1/products/compute/api.yaml

```diff
@@ -12434,13 +12433,13 @@ objects:
         name: 'logConfig'
         description: |
           Configure logging on this health check.
-        min_version: beta
         properties:
             - !ruby/object:Api::Type::Boolean
               name: 'enable'
               description: |
                 Indicates whether or not to export logs. This is false by default,
                 which means no health check logging will be done.
+              default_value: false
```

Comment:
Why would we change this when GAing the fields?

It feels dangerous to change default values for no reason, can we remove this?

Reply:
I'm applying the fix from #3861

Reply:
For me, the fix is more important the promotion

Reply:
Ah, ok it's a region/zone thing that got missed previously. Makes sense